docs/recommendations/DNA/substitution.md

```diff
@@ -2,24 +2,26 @@
 
 ## Definition
 
-Substitution: a sequence change where, compared to a reference sequence, <b>one</b> nucleotide is replaced by <b>one</b> other nucleotide.
+Substitution: a sequence change where, compared to a reference sequence, **one** nucleotide is replaced by **one** other nucleotide.
 
 ## Description
 
-Format:   **"prefix""position_substituted""reference_nucleotide"">"new_nucleotide"**,  e.g. g.123A>G
+Format: `<prefix><position_substituted><reference_nucleotide><new_nucleotide>`
```

Comment:
Yep. This work isn't done. style.html is a sandbox to develop a format that looked good. (I didn't want to update all of them until I was pretty sure that we had consensus on a style.)

Reply:
I like the use of less-than and greater-than around the variables, and I like the spaces and quotes around the characters as in the style guide. When variables have "boundaries" like less-than and greater-than, we can give them spaces instead of underscores. Like:
`<prefix> "." <position substituted> <reference nucleotide> ">" <new nucleotide>`
If we have time in the meeting today, we could discuss this, too.